### ISSUE TITLE
[Feature]: UX/Accessibility: Make fade-in transitions configurable / add "reduce motion" option

### ISSUE THREAD

**Issue by wujekbogdan** [jeffvli/feishin#1553]
### I have already checked through the existing feature requests and found no duplicates

- [x] Yes

### Is this a server-specific feature?

Not server-specific

### What do you want to be added?

The fade-in animation when navigating through the app looks nice, but adds perceived latency. Even if the app loads fast, the transition makes it feel slower.

I understand some users appreciate smooth animations, so keeping them on by default makes sense. But adding an option to disable or reduce them would be great for users who prefer snappier navigation.

Could be a toggle in settings, or even tied to the OS-level `prefers-reduced-motion` media query - which is the recommended approach for accessibility (WCAG).

**Comment by jeffvli** [comment]
I'm still looking into a clean way to implement this, but in the meantime I've reduced the longest transitions:

Page fade: 1s -> 0.3s
List fade: 1s -> 0.5s

**Comment by nomakwae** [comment]
Hello, I am particularly motion sensitive, and to me the most nauseating animation has been hovering over an album, especially if I move my mouse over multiple albums and their controls appear and disappear and move up and down.

For the other people looking for reduced motion:
Electron does provide us with the `--disable-smooth-scrolling` command line option, which already helps a lot.

Some animations are fixable by using custom css:
```css
* {
  animation-duration: 0s !important;
  transition-duration: 0s !important;
  animation-delay: 0s !important;
  transition-delay: 0s !important;
}
```
but these cause some other visual bugs (for example when closing the settings pane, there is a noticeable delayed black overlay that takes a bit to disappear), and do not affect the controls when hovering over an album. I presume this is because they use `motion.div` or `AnimatePresence` from `motion/react`, but do not take my word for it, I'm not a React developer.

For the time being, I've cloned the repo and changed `motion.div` to `div` at multiple places and removed a few instances of `AnimatePresence`, which does help the issue a lot, but I would like this to be officially supported.

Edit: Changing `item-card-controls.tsx` motion.div to div made the app instantly more usable for me
